A55-xds-stateful-session-affinity.md

```diff
@@ -320,27 +323,28 @@ for maintaining session affinity. If the new subchannel goes into
 `TRANSIENT_FAILURE`, we just delegate to the child policy.
 
 
-Note that we unconditionally create the `"override_host_experimental"` policy
-as the child of `“xds_cluster_impl”` even if the feature is not configured
-(in which case, it will be a no-op). The policy's config will have a boolean
-field called `enabled` which will be set by the
-`"cluster_resolver_experimental"` policy based on the boolean in its own
+Note that we unconditionally create the `override_host_experimental` policy
+as the child of `xds_cluster_impl_experimental` even if the feature is not
+configured (in which case, it will be a no-op). The policy's config will have
+a boolean field called `enabled` which will be set by the
+`xds_cluster_resolver_experimental` policy based on the boolean in its own
 config which will be set based on whether the
 [`common_lb_config.override_host_status`][or-host-status] field
 in the CDS resource includes UNKNOWN or HEALTHY. The
-`“cluster_resolver_experimental”` policy receives the CDS update and passes
-the boolean via its child (`“priority_experimental”`) to the
-`“xds_cluster_impl”` policy.
+`xds_cluster_resolver_experimental` policy receives the CDS update and passes
+the boolean via its child (`priority_experimental`) to the
+`xds_cluster_impl_experimental` policy.
 
-If the `"override_host_experimental"` policy is disabled, it should still
+If the `override_host_experimental` policy is disabled, it should still
 maintain the `overrideHostMap`, so that it has all of the necessary data
 to start working if it becomes enabled later, but it does not inject its
 own picker when disabled.
 
 
-One of the existing policies (`"wrr-locality-experimental"`, `"weighted_target"`
-or `"ring_hash_experimental"`) is made as the child policy of
-`“override-host-experimental”` as shown in the following diagram.
+One of the existing policies (`wrr_locality_experimental`,
+`weighted_target_experimental` or `ring_hash_experimental`) is made as
```

Comment:
It won't be `weighted_target`; as of gRFC A52, we'll always use `xds_wrr_locality` instead.

Reply:
Okay. Also you mean `xds_wrr_locality_experimental` ?

Reply:
Yes.